### PR TITLE
Split GPS questions in export

### PR DESCRIPTION
**Pull request by benrudolph**
@NoahCarnahan a little goodie 🍬 
screen shot 2016-04-05 at 5 34 28 pm
cc: @sravfeyn

### CODE REVIEW

File: corehq/apps/export/models/new.py [hunk 1]
```diff
@@ -174,7 +185,7 @@ def create_default_from_export_item(table_path, item, app_ids_and_versions):
             "is_advanced": is_case_update or False,
         }
 
-        if isinstance(item, MultipleChoiceItem):
+        if isinstance(item, MultipleChoiceItem) or isinstance(item, GeopointItem):
```
Comment (NoahCarnahan) [inline corehq/apps/export/models/new.py]: You could replace this with `if isinstance(item, SplitableItemMixin):` I think

Comment (sravfeyn) [inline corehq/apps/export/models/new.py]: Or can also shorten it `isinstance(item, (MultipleChoiceItem, GeopointItem))`.

File: corehq/apps/export/models/new.py [hunk 2]
```diff
@@ -721,6 +753,37 @@ def merge(cls, one, two):
         item.options = options
         return item
 
+    def split_header(self, header, ignore_unspecified_options):
+        header_template = header if '{option}' in header else u"{name} | {option}"
+        headers = []
+        for option in self.options:
+            headers.append(
```
Comment (sravfeyn) [inline corehq/apps/export/models/new.py]: It's possible to render template like strings using Django template engine like below, the logic is somewhat easier to grasp that way.
```python
In [29]: from django.template import Template, Context

In [30]: header = "{% for x in options %}{{ x }}{% endfor %}"

In [31]: Template(header).render(Context({"options":[1,3]}))
Out[31]: u'13'
```
Though, I am not sure if it's favorable

File: corehq/apps/export/tests/test_export_item.py [hunk 3]
```diff
@@ -54,3 +55,19 @@ def test_wrap_export_item_child(self):
         wrapped = ExportItem.wrap(item.to_json())
         self.assertEqual(type(wrapped), type(item))
         self.assertEqual(wrapped.to_json(), item.to_json())
+
+
+def TestGeopointItem(SimpleTestCase):
+
+    def test_split_value(self):
```
Comment (sravfeyn) [inline corehq/apps/export/tests/test_export_item.py]: Could make this more concise by using [`generate_cases`](https://github.com/dimagi/commcare-hq/blob/master/corehq/util/test_utils.py#L259)

**Comment by czue** [comment]
very cool. :earth_africa:

File: corehq/apps/export/models/new.py (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/export/models/new.py]: yeah, definitely prefer that approach with more heavy weight things, but felt like this was sufficiently light weight

File: corehq/apps/export/models/new.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/export/models/new.py]: 👍 will update

**Pull request merged**